recipes/pydoe2/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.1.1" %}
+
+package:
+  name: pydoe2
+  version: {{ version }}
+
+source:
+    fn: pyDOE2-{{ version }}.zip
+    url: https://pypi.io/packages/source/p/pyDOE2/pyDOE2-{{ version }}.zip
+    sha256: c5c0c343bec9b7d0f6650f2d0686802c212ed36ef8b1f895d9cde2ea62768736
+
+build:
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+
+  run:
+    - python
+    - scipy >=0.13
+    - numpy >=1.8
+
+test:
+  imports:
+    - pyDOE2
+
+about:
+  home: https://github.com/clicumu/pyDOE2
+  license: BSD 3-Clause
```

Comment:
please, add the ``license_family`` and also the ``license_file``

Reply:
ok, (opened https://github.com/clicumu/pyDOE2/pull/7)